src/swagger-specs/query-service.yaml

```diff
@@ -675,6 +678,7 @@ paths:
       tags:
         - "Query-Templates"
       summary: Retrieve a query template
+      description: This endpoint retrieves the query template for the `{queryTemplateId}` provided.
```

Comment:
Keep in mind that all path parameters should be stylized in `ALL_CAPS_SNAKE_CASE`. So in this case, the Swagger parameter itself and all mentions of it in descriptions should appear as `{QUERY_TEMPLATE_ID}`.